examples/notebooks/healthcare/beam_nlp.ipynb

```diff
@@ -0,0 +1,968 @@
+      "execution_count": null,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "The following defines a `PTransform` named `ReadLinesFromText`, that extracts lines from a file."
+      ],
+      "metadata": {
+        "id": "dO1A9_WK5lb4"
+      }
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "class ReadLinesFromText(beam.PTransform):\n",
+        "\n",
+        "    def __init__(self, file_pattern):\n",
+        "        self._file_pattern = file_pattern\n",
+        "\n",
+        "    def expand(self, pcoll):\n",
+        "        return (pcoll.pipeline\n",
+        "                | beam.io.ReadFromText(self._file_pattern))"
+      ],
+      "metadata": {
+        "id": "t5iDRKMK5n_B"
+      },
+      "execution_count": null,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "The following sets up an Apache Beam pipeline with the *Interactive Runner*. The *Interactive Runner* is the runner suitable for running in notebooks. A runner is an execution engine for Apache Beam pipelines."
+      ],
+      "metadata": {
+        "id": "HI_HVB185sMQ"
+      }
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "from google.auth import default\n",
+        "\n",
+        "credentials = default()"
+      ],
+      "metadata": {
+        "id": "dMc10Dlgtp1c"
+      },
+      "execution_count": null,
+      "outputs": []
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "p = beam.Pipeline(options = options)"
+      ],
+      "metadata": {
+        "id": "7osCZ1om5ql0"
+      },
+      "execution_count": null,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "The following sets up a PTransform that extracts words from a Google Cloud Storage file that contains lines with each line containing a In our example, each line is a medical notes excerpt that will be passed through the Healthcare NLP API\n",
+        "\n",
+        "| is an overloaded operator that applies a PTransform to a PCollection to produce a new PCollection. Together with |, >> allows you to optionally name a PTransform.\n",
+        "\n",
+        "Usage: [PCollection] | [PTransform] or [PCollection] | [name] >> [PTransform]"
+      ],
+      "metadata": {
+        "id": "EaF8NfC_521y"
+      }
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "lines = p | 'read' >> ReadLinesFromText(GCS_BUCKET + \"nlpsample500.csv\")"
+      ],
+      "metadata": {
+        "id": "2APAh6XQ6NYd"
+      },
+      "execution_count": null,
+      "outputs": []
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "if debug:\n",
+        "    ib.show(lines)"
+      ],
+      "metadata": {
+        "id": "NH_04EXw6P8F"
+      },
+      "execution_count": null,
+      "outputs": []
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "if debug:\n",
```

Comment:
There's a lot of consecutive code blocks here with not much explanation as to what they're doing, I think dropping some text snippets in here would help.